oshi-core/src/main/java/oshi/hardware/GlobalMemory.java

```diff
@@ -36,14 +36,14 @@
      *
      * @return Total number of bytes.
      */
-    long getTotal();
+    long getMemTotal();

```

Comment:
It's an abbreviation on a private variable that an automated getter/setter-maker produced. :)

But seriously, I was leaving the option open of changing variable names. "Physical" and "Virtual" would represent the two types of memory this class covers.   But unless we do something significant like that, there's no great need to change.